src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridge.java

```diff
@@ -549,7 +566,53 @@ private boolean isReady() {
         return this.isReady;
     }
 
-    private void startInactiveConsumerDeletionTimer(Long timeout) {
-        // TODO: to be used a Quarkus timer for scheduling the task
+    private void startInactiveConsumerDeletionTimer(long timeout) throws SchedulerException {
+        long timeInMs = timeout * 1000L;
+        JobDetail job = JobBuilder.newJob(InactiveConsumerDeletionJob.class).build();
+        Trigger trigger = TriggerBuilder.newTrigger()
+                .withIdentity("myTrigger", "myGroup")
```

Comment:
can we use better names for "myTrigger" and "myGroup"?

Reply:
"triggerInactiveConsumerDeletion" for `myTrigger` and `staleConsumers` for `myGroup` ?

Reply:
fine with me :-)